### PR TITLE
Add another solution that compiles to iOS

### PR DESCRIPTION
**Pull request by kditraglia**

### CODE REVIEW

File: TowerDefense/TowerDefense.cs [hunk 1]
```diff
@@ -195,7 +195,7 @@ private void HandleMouse()
             {
                 if (mouse.MouseState.LeftButton == ButtonState.Pressed)
                 {
-                    Exit();
+                    //Exit();
```
Comment (kditraglia) [inline TowerDefense/TowerDefense.cs:198]: iOS games can't call exit on themselves, so I had to remove this

Comment (d0nutptr) [reply TowerDefense/TowerDefense.cs:198]: What are they supposed to do instead?

Comment (kditraglia) [reply TowerDefense/TowerDefense.cs:198]: iOS the user controls when you close the app, you never have the ability to terminate your own app.  I guess you could just divide by zero or something.

Comment (d0nutptr) [reply TowerDefense/TowerDefense.cs:198]: Lol, alright. Maybe in the future just opt for a end game screen.

**Comment by kditraglia** [comment]
I think I'm good to merge this in, I made TowerDefense be a library, and program.cs moves to a project called TowerDefenseWindows.  This shouldn't break any other branch since the paths to all the files is still the same except program.cs which has basically no code in it.

**Pull request merged**